data/gegno/engines.txt

```diff
@@ -0,0 +1,225 @@
+outfit "RG15 Torch Thruster"
+	category Engines
+	licenses
+		"Gegno Civilian"
+	cost 26000
+	mass 31
+	thumbnail "outfit/small torch"
+	"engine capacity" -31
+	"outfit space" -31
+	thrust 7.67
+	"thrusting energy" 0.9
+	"thrusting heat" 2
+	"flare sprite" "effect/ion torch/small"
+		"frame rate" 2.5
+	"flare sound" "ion torch small"
+	description "The RG15 Ion Torch Thruster is the smallest torch thruster found amongst Gegno Vi ships and it is almost exclusively mounted on the Slate. It's not amazingly efficient, weighing more and producing more heat than most thrusters of a similar size, but also has a slightly lower energy consumption rate."
+
+outfit "RG15 Torch Steering"
+	category Engines
+	licenses
+		"Gegno Civilian"
+	cost 21000
+	mass 25
+	thumbnail "outfit/small torch steer"
+	"engine capacity" -25
+	"outfit space" -25
+	turn 205
+	"turning energy" 0.4
+	"turning heat" 1.4
+	"steering flare sprite" "effect/ion torch/small"
+		"frame rate" 2.4
+	"steering flare sound" "ion torch small"
+	description "This steering system that incorporates RG15 thrusters is only meant for the smallest of Gegno Vi ships, and mounting it on anything larger than a Shuttle-class ship is unheard of. Despite this, its energy requirement is a notch lower than normal, so some Gegno pilots use it as a cheap addon."
+
+outfit "RG18 Torch Thruster"
+	category Engines
+	licenses
+		"Gegno Civilian"
+	cost 58000
+	mass 51
+	thumbnail "outfit/med torch"
+	"engine capacity" -51
+	"outfit space" -51
+	thrust 14.8
+	"thrusting energy" 1.2
+	"thrusting heat" 3.5
+	"flare sprite" "effect/ion torch/medium"
+		"frame rate" 2.4
+	"flare sound" "ion torch medium"
+	description "Ion Torch Thrusters are the Gegno's equivalent of humanity's ion thrusters, but are still earlier in design and lacking in most departments when compared. They have heavier frames and their thrust output is a lot more chaotic and explosive, causing them to run notably hotter but require slightly less energy. The RG18 is the most common-sized thruster amongst Gegno Vi vessels, and a notable size at that."
+
+outfit "RG18 Torch Steering"
+	category Engines
+	licenses
+		"Gegno Civilian"
+	cost 46000
+	mass 39
+	thumbnail "outfit/med torch steer"
+	"engine capacity" -39
+	"outfit space" -39
+	turn 420
+	"turning energy" 1.3
+	"turning heat" 2.4
+	"steering flare sprite" "effect/ion torch/medium"
+		"frame rate" 2.4
+	"steering flare sound" "ion torch medium"
+	description "The RG18 Steering system is decently hefty for a steering system of its size. Most mid-ranged Gegno Vi ships run these as a minimum to be able to turn their already heavy ships, and they perform their job just well enough."
```

Comment:
It seemed to come off as arbitrary, but now that you mention it, hefty really only refers to size. How about changing up the first line to be something like,:

"The RG18 steering system is rather robust for a steering system of its size. Most....." ?

Reply:
The change in energy and heat efficieny is remnants of the thrusters being based off of human plasma thrusters as I had used those for comparison. I'm not sure if the human ones do the same thing. It would make sense that a low-tech attempt at their first plasma engines would read as this way, so while it's not intentional, perhaps handling of the scale of things is still an issue of the Gegno? (Same is said elsewhere for a large battery, I believe.)

If not this, what do you suggest?

Reply:
Yeah, one way or another, the idea that their plasma efficiency in relation to scale is not very good. While the Vi have a hard time compacting their technology, the Scin have a hard time doing the opposite.